.veracode-exclusions

```diff
@@ -35,3 +35,4 @@ engine/src/simumod/coder.pl
 grpc/centreon-rpc-client.py
 tests
 packaging
+/root/.conan/
```

Comment:
protobuf generates source code files from our *.proto files of the form *.pb.cc and *.pb.h
I think these files should also been excluded.

Reply:
currently the script cannot use wildcards to exclude files.
if more exclusions are needed, you will be able to add them to this file, after the merge or right now 👍